doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -632,6 +632,12 @@ Modem libraries
 
   * Rename the nRF91 socket offload layer from ``nrf91_sockets`` to ``nrf9x_sockets`` to reflect that the offload layer is not exclusive to the nRF91 Series SiPs.
 
+  * Updated:
+
+    * The :ref:`modem_trace_module` to let the trace thread sleep when the modem trace level is set to off using the `nrf_modem_lib_trace_level_set()` function, and the trace backend returns ``-ENOSPC``. The trace thread wakes up when another trace level is set.
```

Comment:
```suggestion
    * The :ref:`modem_trace_module` to let the trace thread sleep when the modem trace level is set to :kconfig:option:`NRF_MODEM_LIB_TRACE_LEVEL_OFF` using the `nrf_modem_lib_trace_level_set()` function, and the trace backend returns ``-ENOSPC``. The trace thread wakes up when another trace level is set.
```

A bit more precise.

Reply:
I'll use :c:enum`NRF_MODEM_LIB_TRACE_LEVEL_OFF` The Kconfig option can only be used to select the start trace level, while here we want to turn it of when the application requires.

Reply:
Freudian slip, I meant `enum`, yes.